content/en/docs/components/pipelines/v2/run-a-pipeline.md

```diff
@@ -4,77 +4,101 @@ description = "Execute a pipeline on the KFP backend"
+## Run a pipeline from the KFP SDK client <a id="run_from_sdk_client"></a>
 
-```python
-client.create_run_from_pipeline_func('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
-```
+You can programmatically submit a pipeline run from the KFP SDK client in two ways:
+
+* Submit an IR YAML
+* Submit a Python pipeline function for execution
+
+To submit an IR YAML for execution:
+
+1. Instantiate a `Client` by specifying the `host` URL of your KFP instance:
+    ```python
+    from kfp.client import Client
+    client = Client(host='<YOUR_HOST_URL>')
+    ```
+    For more information about the `Client` constructor and method parameters, see the  [kfp.client][kfp-sdk-api-ref-client] in the [KFP SDK API reference][kfp-sdk-api-ref].
+
+1. Use the `.create_run_from_pipeline_package` method:
+
+    ```python
+    client.create_run_from_pipeline_package('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
+    ```
+
+To submit a Python pipeline function for execution:
+
+1. Instantiate a `Client` by specifying the `host` URL of your KFP instance:
+    ```python
+    from kfp.client import Client
+    client = Client(host='<YOUR_HOST_URL>')
+    ```
+
+1. Use the `.create_run_from_pipeline_func` convenience method. This wraps the compilation and run submission into one method.
+
+    ```python
+    client.create_run_from_pipeline_func('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
+    ```
 
-See the [KFP SDK Client reference documentation][kfp-sdk-api-ref-client] for a detailed description of the `Client` constructor and method parameters.
+## Run a pipeline from the KFP SDK CLI <a id="run_from_sdk_cli">
 
-## 3. Run from the KFP SDK CLI
-The `kfp run create` command allows you to submit a pipeline from the command line. `kfp run create --help` shows that this command takes the form:
+You can submit a pipeline from the KFP SDK CLI using the `kfp run create` command. To run this command, use the following syntax:
 
 ```shell
 kfp run create [OPTIONS] [ARGS]...
 ```
 
-For example, the following command submits the `path/to/pipeline.yaml` IR YAML to the KFP backend:
+For example, run the following command to submit the IR YAML file at `path/to/pipeline.yaml` to the KFP backend:
 
 ```shell
 kfp run create --experiment-name my-experiment --package-file path/to/pipeline.yaml
 ```
 
-For more information about the `kfp run create` command, see the [KFP Command Line Interface reference documentation][kfp-run-create-reference-docs]. For more information on the KFP CLI generally see [Command Line Interface user docs][kfp-cli].
+For more information about the `kfp run create` command, see [create][kfp-run-create-reference-docs] in the [KFP SDK API reference][kfp-sdk-api-ref]. For more information on the available commands in the KFP CLI, see [Command Line Interface][kfp-cli] in the [KFP SDK API reference][kfp-sdk-api-ref].
```

Comment:
nit:
`see [create]` -> `see [run create]`

The `create` action can be used for multiple resources, so probably want to be specific here.